guides/source/association_basics.md

```diff
@@ -735,12 +735,10 @@ a.first_name = 'David'
 a.first_name == b.author.first_name # => true
 ```
 
-Active Record supports automatic identification for most associations with standard names. However, Active Record will not automatically identify bi-directional associations that contain any of the following options:
+Active Record supports automatic identification for most associations with standard names. However, Active Record will not automatically identify bi-directional associations that contain a scope or any of the following options:
 
 * `:conditions`
```

Comment:
Just spotted in passing, but.. didn't we stop supporting `:conditions` options on associations in 4.x? 🙊

Reply:
Good catch!